### DISCUSSION TITLE
PCB repair

### DISCUSSION THREAD

**Discussion by hanqiuli** [bitcraze/discussions#1095] [Q&A]
Hi all, I have accidentally powered the Crazyflie with a power supply above the rated voltage (6V). As a result the PCB was bricked... Is this repairable at all? Has anyone had this issue before?

**Comment by tobbeanton** [comment]
If something broke it is probably the power management chip BQ24075. It requires hot-air to replace and some soldering experience.

You can try and power it using a current limiting power supply (set ~200mA as limit) at 4-5V at the VCOM expansion port pin. This is after the BQ24075 and if it works then you know the PMC is the problem.
